Examples/Continuous/HarmonicOscillator/HO.py

```diff
@@ -0,0 +1,30 @@
+import netket as nk
+
+import jax.numpy as jnp
+import netket.operator
+
+
+def v(x):
+    return jnp.linalg.norm(x) ** 2
+
+
+L = 10.0
+
+hilb = nk.hilbert.Particle(N=5, L=(jnp.inf,), pbc=(False,))
+
+sab = nk.sampler.MetropolisGaussian(hilb, sigma=1.0, n_chains=16, n_sweeps=1)
+model = nk.models.Gaussian(dtype=float)
+
+ekin = nk.operator.KineticEnergy(hilb, 5 * [2.0])
```

Comment:
why 5*[2.0] ?

Reply:
I'll replace with mass=2. 
But in this way one can see that it is possible to give a list ergo different values for the mass. Even though not explicitly shown.